### PR TITLE
[CORL-2729] Moderation Queue Focus 

### PR DESCRIPTION
**Pull request by marcushaddon**
## What does this PR do?
This PR fixes a bug in which upon navigating to `/admin/moderate`, focus immediately shifts to the first moderation card, which is an anti-pattern for accessibility. 

## These changes will impact:

- [ ] commenters
- [x] moderators
- [x] admins
- [ ] developers

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Ensure that there are multiple comments in the moderation queue
2. As an admin or moderator, navigate to `/admin/moderate`
3. Press the tab key
4. Observe that the focus begins at the first element on the page (the nav at the top) and cycles through the different nav options, rather than skipping them and jumping to the beginning of the mod queue
5. Observe that using the "j" and "k" keys still cycles through mod cards
 
 
## How do we deploy this PR?
No special considerations should be necessary

### CODE REVIEW

File: src/core/client/admin/routes/Moderate/Queue/Queue.tsx [hunk 1]
```diff
@@ -99,6 +109,28 @@ const Queue: FunctionComponent<Props> = ({
...
   }, [window.document]);
 
+  useEffect(() => {
+    key(HOTKEYS.NEXT, QUEUE_HOTKEY_ID, selectNext);
+    key(HOTKEYS.PREV, QUEUE_HOTKEY_ID, selectPrev);
+
+    // key(HOTKEYS.BAN, QUEUE_HOTKEY_ID, onBan);
+
+    // key(HOTKEYS.APPROVE, QUEUE_HOTKEY_ID, onApprove);
+    // key(HOTKEYS.REJECT, QUEUE_HOTKEY_ID, onReject);
```
Comment (nick-funk) [inline src/core/client/admin/routes/Moderate/Queue/Queue.tsx]: are these from past debugging or do they need to be uncommented?

File: src/core/client/admin/routes/Moderate/Queue/Queue.tsx [hunk 2]
```diff
@@ -99,6 +109,28 @@ const Queue: FunctionComponent<Props> = ({
...
+
+    // The the scope such that only events attached to the ${id} scope will
+    // be honored.
+    key.setScope(QUEUE_HOTKEY_ID);
+
+    return () => {
+      // Remove all events that are set in the ${id} scope.
+      key.deleteScope(QUEUE_HOTKEY_ID);
+    };
+    // key.deleteScope(id);
```
Comment (nick-funk) [inline src/core/client/admin/routes/Moderate/Queue/Queue.tsx]: old debug code?

**Review by nick-funk — CHANGES_REQUESTED** [review]

File: src/core/client/admin/routes/Moderate/Queue/Queue.tsx (hunk 2 above)
Comment (marcushaddon) [reply src/core/client/admin/routes/Moderate/Queue/Queue.tsx]: ✅

**Review by nick-funk — APPROVED** [review]
Looks good! Nice work!

**Pull request merged**